OG_Format.adoc

```diff
@@ -21,16 +21,16 @@ From there the working group organized regular meetings to validate the evolutio
 ////
 == Authors
 
-* https://github.com/castelao[Guilherme Castelão], _Scripps Institution of Oceanography - UC San Diego, USA_
-* https://github.com/vturpin[Victor Turpin], _OceanGliders Technical Coordinator at OceanOPS, Brest, France_
-* https://github.com/kerfoot[John Kerfoot], _Institute of Marine & Coastal Sciences, Rutgers University, New Jersey, USA_
-* https://github.com/soerenthomsen[ Sören Thomsen ], _LOCEAN, ISPL, Sorbonne University, Paris, France,_
+* https://github.com/castelao[Guilherme Castelão] (lead), _Scripps Institution of Oceanography - UC San Diego, USA_
```

Comment:
I suggest everyone in the leading team be in alphabetical order.

Reply:
Agree.